src/components/GoodsList.tsx

```diff
@@ -0,0 +1,15 @@
+import { Good } from './Good';
+
+type GoodsListProps = {
+  goods: string[];
+};
+
+export const GoodsList: React.FC<GoodsListProps> = ({ goods }) => {
+  return (
+    <ul>
+      {goods.map((good, index) => (
+        <Good goodName={good} key={index} />
```

Comment:
Using the array index as a `key` prop is not recommended as it can negatively impact performance and may cause issues with component state. Instead, use a unique identifier like an ID from the data. [CODE KNOWLEDGE]